app/models/manageiq/providers/ibm_cloud/inventory/collector/vpc.rb

```diff
@@ -0,0 +1,19 @@
+class ManageIQ::Providers::IbmCloud::Inventory::Collector::VPC < ManageIQ::Providers::IbmCloud::Inventory::Collector
+  require_nested :CloudManager
+
+  def connection
+    manager.connect
+  end
+
+  def vms
+    connection.instances.all
+  end
+
+  def images
+    connection.images.all
+  end
+
+  def image(image_id)
+    connection.images.instance(image_id).details
```

Comment:
If this can return `nil` you might want to `&.details` so it doesn't raise an exception if it can't e.g. find that image by id (got deleted between calling `connection.images.all` and calling `connection.images.instance(image_id)` for example)